packages/manager/apps/hpc-vmware-managed-vcd/mocks/vcd-organization/vcd-catalog.mock.ts

```diff
@@ -0,0 +1,156 @@
+import { TVcdCatalog } from '../../src/types/vcd-catalog.interface';
+
+export const catalogProductList: TVcdCatalog = [
+  {
+    productName: 'pack-3264',
+    mandatory: false,
+    exclusive: false,
+    family: 'host_order',
+    productType: 'cloud_service',
+    planCode: 'pack-3264',
+    prices: [
+      {
+        priceInUcents: 0,
+        pricingType: 'rental',
+        maximumRepeat: 1,
+        price: { currencyCode: 'EUR', text: '0.00 €', value: 0 },
+        minimumRepeat: 1,
+        capacities: ['installation'],
+        duration: 'P0D',
+        maximumQuantity: null,
+        interval: 0,
+        description: 'Installation pricing',
+        pricingMode: 'default',
+        minimumQuantity: 1,
+      },
+      {
+        pricingMode: 'default',
+        minimumQuantity: 1,
+        description: 'Installation pricing',
+        interval: 1,
+        maximumQuantity: null,
+        duration: 'P1M',
+        capacities: ['renew'],
+        minimumRepeat: 1,
+        price: { value: 80, text: '80.00 €', currencyCode: 'EUR' },
+        maximumRepeat: null,
+        pricingType: 'rental',
+        priceInUcents: 8000000000,
```

Comment:
peut-être utiliser le format de chiffres proposé par Thibs ?
```suggestion
        priceInUcents: 8_000_000_000,
```